holidayextras/brands
# Modals - Global XL version and Paultons/Peppa modals

#### What does this PR do? (please provide any background)
Moves up the XL version of the modal sizing into global and concentrates on moving the Paultons/Peppa Modals into this new container, whilst returning default md/lg variants of modals back to bootstrap default widths

#### How can this be tested?
`cd pattern-library && npm link ../brands/brands/paultons`
`cd pattern-library && npm link ../brands/brands/peppa`
view paultons or peppa in http://patternlibrary.holidayextras.com/playground/shortbreaks-more-info/index.html to see xl variant in action

#### Any tech debt?
I don't believe so

#### Screenshots / Screencast


#### What gif best describes how you feel about this work?
![debugging](https://cloud.githubusercontent.com/assets/3897844/22929307/da72dab8-f2b2-11e6-9627-e814fe9c34ea.gif)



---

**Reviewer**

By adding a +1 you are confirming you have...
- Witnessed the work behaving as expected (this could be on the author's machine or screencast).
- Checked all the work locally on pattern library.

---
